src/Message.js

```diff
@@ -42,6 +43,17 @@ export default class Message extends React.Component {
     return <Bubble {...bubbleProps}/>;
   }
 
+  renderSystemMessage() {
+    if (this.props.currentMessage.system) {
```

Comment:
Might remove this line, because it is already checked on https://github.com/FaridSafi/react-native-gifted-chat/pull/608/files#diff-14cb4b4f30af078adef32e3aa08999eaR73